### PR TITLE
feat:Badge

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
https://user-images.githubusercontent.com/86787002/154591715-4e159846-1156-44d4-b8c3-87ed537bcf79.mp4

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.badge {
+    position: relative;
+    width: max-content;
+    min-width: 2.5rem;
+    font-size: 2rem;
+    margin: 0 1rem;
+}
+
+.badge>div {
```
Comment (lannisterrr) [inline Components/Badge/badge.css:16]: don't target the elements directly , give a class to that particular div.

File: Components/Badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+            <li><a href="https://github.com/Shivanshsharma01/Mango-UI">Github <i class="fa-brands fa-github"></i></a>
+            </li>
+
+        </ul>
+    </nav>
+    <div class="container">
+
+
+        <div class="side-content">
+            <div class="sidebar">
```
Comment (lannisterrr) [inline Components/Badge/badge.html]: you can use semantic HTML's aside tag here

File: Components/Badge/badge.html [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+            <li><a href="index.html">Home <i class="fa-solid fa-house-chimney"></i></a></li>
+            <li><a href="https://github.com/Shivanshsharma01/Mango-UI">Github <i class="fa-brands fa-github"></i></a>
+            </li>
+
+        </ul>
+    </nav>
+    <div class="container">
+
+
+        <div class="side-content">
```
Comment (lannisterrr) [inline Components/Badge/badge.html]: you can use semantic HTML's  "section" tag here

File: Components/Badge/badge.html [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+    <link rel="stylesheet" href="badge.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+
+    <title>Mango UI</title>
+</head>
+
+<body>
+    <nav>
```
Comment (lannisterrr) [inline Components/Badge/badge.html]: It's good to wrap the main nav in a "header" tag.

File: Components/Badge/badge.html (hunk 2 above)
Comment (Shivanshsharma01) [reply Components/Badge/badge.html:34]: This div is for whole main content, maybe aside tag will not be appropriate for this

File: Components/Badge/badge.html [hunk 5]
```diff
@@ -14,24 +14,29 @@
...
-        </div>
+                <a href="#"> Mango UI <i class="fa-solid fa-mango"></i> </a>
+            </div>
+
+            <ul class="nav-pills">
+                <li><a href="index.html">Home <i class="fa-solid fa-house-chimney"></i></a></li>
+                <li><a href="https://github.com/Shivanshsharma01/Mango-UI">Github <i
+                            class="fa-brands fa-github"></i></a>
+                </li>
+
```
Comment (saadahmadhere) [inline Components/Badge/badge.html:29]: here is an extra line space

File: Components/Badge/badge.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="badge.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+
+    <title>Mango UI</title>
```
Comment (MaruthiPothuganti) [inline Components/Badge/badge.html]: here you can distinguish each page of component  for ex. **Badge | MangoUI**

File: Components/Badge/badge.html [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+                </div>
+                <div class="content-subheading">
+                    Badges are usually seen in social media apps to indicate the status of whether a person is online or
+                    offline. They are also used to indicate number of unread notifications. There are multiple variants
+                    of these that come with different colors and
+                </div>
+
+                <div class="divider"></div>
+
+
```
Comment (saadahmadhere) [inline Components/Badge/badge.html:126]: too much space here

**Review by saadahmadhere — APPROVED** [review]
Overall code looks clean, nice use of variables. 
Using prettifier would take care of the spacing issues itself.

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested one change. Everything else looks good.

File: Components/Badge/badge.html (hunk 6 above)
Comment (Shivanshsharma01) [reply Components/Badge/badge.html]: sure , thanks

File: Components/Badge/badge.html (hunk 7 above)
Comment (Shivanshsharma01) [reply Components/Badge/badge.html:126]: using beautify still it's not reducing the space may be a glitch. Thanks for the suggestion

**Pull request merged**